fe/src/project/AspNetCore/areas/DownloadCenter/components/DownloadCenterCatalog/DownloadCenterCatalog.js

```diff
@@ -0,0 +1,44 @@
+import React, { useState } from "react";
+import PropTypes from "prop-types";
+
+const DownloadCenterCatalog = (props) => {
+    const [items] = useState(props.pagedResults.data ? props.pagedResults.data : []);
+
+    return (
+        <div className="section dc-catalog">
+            <div className="container">
+                <div className="columns">
+                    <div className="column">
+                        {items ? (
+                            items.map((item, index) => {
+                                return (
+                                    <div className="dc-catalog__container" key={index}>
+                                        <h3 className="is-size-5 has-text-weight-bold is-uppercase">{item.categoryName}</h3>
+                                        <div className="is-flex is-flex-wrap-wrap mt-2">
+                                            {item.categories.length > 0 && item.categories.map((category, index) => {
```

Comment:
**suggestion:** item.categories can be null. It should be:
{item.categories && item.categories.length > 0 && item.categories.map